tests/neo4j/test_tx_func_run.py

```diff
@@ -214,9 +221,9 @@ def update2(tx):
             "w", bookmarks=self._session1.last_bookmarks(), database=db
```

Comment:
maybe for another PR since it's not related to the change, but why use 2 sessions and manually chain bookmarks when we could use 1 session instead?

Reply:
I need two sessions because I'm using two managed transactions at the same time to make sure the server is holding the lock on a node while trying to update that node with another managed tx (i.e., another session.)